### ISSUE TITLE
Should we disable invalid variable setters and getters?

### ISSUE THREAD

**Issue by mark-friedman** [mit-cml/blockly-plugins#3]
Maybe in addition to setting the warning we should disable the block.

**Comment by Nishantware** [comment]
Yes, you should generally disable invalid variable setters and getters, as it is considered good practice to prevent setting values that could put your object in an inconsistent or invalid state by implementing validation within the setter methods, effectively guarding against incorrect data being assigned to your variables.

**Comment by TannerGabriel** [comment]
I believe this would be a good addition since otherwise the blocks would still be used in the code generation even if the variable is incorrect. I will provide a quick fix for this.
